src/views/EventPage.vue

```diff
@@ -86,22 +87,31 @@ export default {
       }
     }
   },
-  props: {
-    event: Object,
-  },
   data() {
     return {
       // navigation component options
       navOptions: {
         topType: 'share-light',
         backRoute: '/discover'
       },
-      snipped: true
+      snipped: true,
+
+      backgroundImg: null,
+      numPhotos: 0,
+      imageList: [],
+      tags: [],
+      eventName: '',
+      venueName: '',
+      startDate: '',
+      description: '',
```

Comment:
to sort of wrap it this way?
```suggestion
      event: {
          backgroundImg: null,
          numPhotos: 0,
          imageList: [],
          tags: [],
          eventName: '',
          venueName: '',
          startDate: '',
          description: '',
      }
```